docs/src/drivers.md

```diff
@@ -0,0 +1,19 @@
+# Drivers
+
+| Name                                                                           | COSI Driver Name                                                                       | Description                                                                                                                | Compatible with COSI Version(s) |
+| ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- | ------------------------------- |
+| [Akamai Cloud Object Storage](https://www.linode.com/products/object-storage/) | [`objectstorage.cosi.linode.com`](https://github.com/linode/linode-cosi-driver)        | A Kubernetes Container Object Storage Interface (COSI) Driver for Linode                                                   | `v1alpha1`                      |
+| [Azure Blob](https://azure.microsoft.com/en-us/products/storage/blobs)         | [`blob.cosi.azure.com`](https://github.com/Azure/azure-cosi-driver)                    | This driver allows Kubernetes to use Azure Blob Storage using the Container Storage Object Interface (COSI) infrastructure | `v1alpha1`                      |
+| [Ceph Rados Gateway](https://docs.ceph.com/en/latest/radosgw/)                 | [`ceph.objectstorage.k8s.io`](https://github.com/ceph/ceph-cosi)                       | COSI driver for Ceph Object Store aka RGW                                                                                  | `v1alpha1`                      |
+| [Dell ObjectScale](https://www.dell.com/en-us/dt/storage/objectscale.htm)      | [`cosi.dellemc.com`](https://github.com/dell/cosi)                                     | COSI Driver for Dell ObjectScale                                                                                           | `v1alpha1`                      |
```

Comment:
| [HPE Alletra Storage MP X10000](https://www.hpe.com/us/en/alletra-storage-mp-x10000.html) | [`cosi.hpe.com`](https://github.com/hpe-storage/cosi-driver) | A Kubernetes Container Object Storage Interface (COSI) driver for HPE Alletra Storage MP X10000 | `v1alpha1` |